tests/ui/cypress/specs/multi_user_openid_connect.js

```diff
@@ -138,7 +176,90 @@ describe('OpenID Connect users', () => {
     cy.get('input[name="username"]')
   })
 
-  it('Login receives short-lived refreshable token', () => {
+  it('Login with short-lived non-refreshable token and try to create a CA', () => {
```

Comment:
I'm confused about this test. I have a feeling of deja vu like we discussed this before, sorry if that's the case. A diff of this test with the test above seems to only show that this one is a subset of the one above, as the one above also calls `visit('/')` at the end and this one does not. The other differences are that the test above uses updated lookup selectors and this test does not. It seems to me like this test should be removed and keep only the one above, but I have the feeling you said there was a valuable difference, but I can't see it at the moment. What am I missing?

Reply:
The one above doesn't need to try to create a CA (doesn't hurt either?), think, but the refresh is worthwhile to test: If the token has expired and can't be refreshed, a refresh should _not_ show you this page, but the OIDC login page.

But yes, at least I could used the cy.contains function. updated.